components/nextBlock/nextBlock.tsx

```diff
@@ -1,24 +1,24 @@
-import React from 'react'
-import Block, { BlockColor } from '../block/block'
-import styles from './nextBlock.module.scss'
+import React, { useContext } from "react";
+import Block from "../block/block";
+import styles from "./nextBlock.module.scss";
+import { shapes } from "../../utils";
+import { AppContext } from "../../context/state";
 
 // Draws the "next" block view showing the next block to drop
 const NextBlock = () => {
+  const { state, dispatch } = useContext(AppContext);
 
-    const box = [[0,0,0, 0]];
+  const nextShape = state.nextShape;
+  const box = shapes[nextShape][0]; // Get the first rotation
 
-    // Map the block to the grid
-    const grid = box.map((rowArray, row) => {
-        return rowArray.map((col) => {
-            return <Block key={`${row}${col}`} blockColor={BlockColor.pink}/>
-        })
-    })
+  // Map the block to the grid
+  const grid = box.map((rowArray, row) => {
+    return rowArray.map((index, col) => {
+      return <Block key={`${row}${col}`} color={index} />;
```

Comment:
the index of the 'shapes' array (in utils- the array that holds all the various shapes and their rotations) defines the color of the block

Reply:
Yeah this makes way more sense if the shape is always the same color, good call